### PR TITLE
Library: Port 'Color Picker' to Python

### PR DESCRIPTION
**Pull request by gregorni**
Doesn't work yet. Somehow `Gdk.RGBA` doesn't properly set the color values, only to `(0,0,0,0)`

### CODE REVIEW

File: src/Library/demos/Color Picker/main.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+import workbench
+
+portal = Xdp.Portal()
+parent = XdpGtk4.parent_new_gtk(workbench.window)
+button = workbench.builder.get_object("button")
+
+
+def on_selected(_portal, task):
+    r, g, b = _portal.pick_color_finish(task)
+    print(r, g, b)
```
[the remark this answers is not in the document]
Comment (gregorni) [reply src/Library/demos/Color Picker/main.py:16]: Oh, yes, woops 😬

**Comment by theCapypara** [comment]
The PyGObject bindings for RGBA do not take arguments anymore. See: https://gitlab.gnome.org/GNOME/pygobject/-/issues/560#note_1607269

**Comment by gregorni** [comment]
@theCapypara Thanks, that was the problem. My latest commit fixes the problem, although it somehow feels a bit weird, not sure if it's deviating too much from the JS version.

**Review by theCapypara — APPROVED** [review]
LGTM! The way you assign to RGBA now looks fine to me as well. Thank you!

**Pull request merged**